web/src/beta/lib/core/engines/Cesium/hooks.ts

```diff
@@ -370,6 +374,11 @@ export default ({
   const handleClick = useCallback(
     async (e: CesiumMovementEvent, target: RootEventTarget) => {
       mouseEventHandles.click?.(e, target);
+
+      if (!(featureFlags & MAP_FEATURE_FLAGS.SINGLE_SELECTION)) return;
+
+      // === Layer selection ===
```

Comment:
Is this necessary? If it was on purpose, it seems like you have a couple different comment styles in this file (better if core has all the same style. Even better if ReEarth and core share the same style.).
If it was just forgotten, please remove

Reply:
Then can you keep the style the same as elsewhere? 🙏🏻 Everywhere else is just the simple `// Comment goes here`

Reply:
Fixed! [Update hooks.ts](https://github.com/reearth/reearth/pull/507/commits/18c6ff9cb5e8f3263342130d618d9d2de1e8000d)